site2/docs/concepts-messaging.md

```diff
@@ -411,4 +411,32 @@ Message deduplication makes Pulsar an ideal messaging system to be used in conju
 
 > More in-depth information can be found in [this post](https://streaml.io/blog/pulsar-effectively-once/) on the [Streamlio blog](https://streaml.io/blog)
 
+## Delayed Message Delivery
+Delayed Message Delivery enables you to consume new message later, instead of immediately as normal. In this mechanism, messages is stored in memory after publish to broker, after specific delayed time passed, the message will be delivered to consumer.  
```

Comment:
Messages is stored in bookkeeper,  DelayedDeliveryTracker maintain the time index(time -> messageId) in memory.

Reply:
@codelipenghui done